src/styles/utilities/sizing.ts

```diff
@@ -59,4 +59,4 @@ export default {
     wAuto: {
         width: 'auto',
     },
-};
+} as const;
```

Comment:
@JKobrynski This is quite recent but let's try to follow this pattern for style files -> https://github.com/Expensify/App/pull/25755/files#r1302971220

For your case you will probably have to use `ViewStyle`.

Reply:
Done!